csharp/HOLMS.Platform/HOLMS.Platform/Support/DTOBuilders/Booking/Groups/GroupBookingBuilder.cs

```diff
@@ -34,10 +32,8 @@ public class GroupBookingBuilder {
         public GroupBookingBuilder(GroupBookingIndicator id, GroupBookingStatus status,
             bool taxExempt, string taxId, bool groupPaysLodging, bool groupPaysIncidentals,
             string additionalNotes, string customerBookingId, InclusiveOpsdateRange bookingDates,
-            AccountIndicator houseAccount, RateScheduleIndicator rs, GroupIndicator g,
-            Guid? groupBookingMethodId,
-            Guid? reservationSourceId, Guid? travelAgentId,
-            Guid? cancellationPolicy,
+            RateScheduleIndicator rs, GroupIndicator g, Guid? groupBookingMethodId,
```

Comment:
Derek, just FYI, I think the plan here is (1) to define a model-level "channel" concept inside the booking context, (2) explicitly attribute each reservation to a channel, (3) move some things, including cancellation policy, correspondence policy, and perhaps others to the channel, and (4) clarify and articulate the relationship between group booking and channel. 

A group booking may be a "pseudo-channel" in that it has settable cancellation policy and other terms.

Don't worry about it now, just a glimpse of what's coming later. We'll probably create the channel over the coming month, one for each of direct-web, direct-walkin, direct-call, direct-group, indirect-booking.com, and indirect-expedia.